packages/@aws-cdk/aws-ec2/test/integ.vpc.expected.json

```diff
@@ -78,7 +75,10 @@
         "GatewayId": {
           "Ref": "MyVpcIGW5C4A4F63"
```

Comment:
Kind of expected to see this -1 here somewhere

Reply:
You're right. Stale build probably.